tools/doc/checkLinks.js

```diff
@@ -1,3 +1,7 @@
+// You are probably here because you failed a PR and wants to see if this thing
+// has a --fix option.
+// It doesn't. But if you have some Unix utilities, you can just feed the last
+// lines of your markdown file to `sort`.
```

Comment:
While that's a valid point, this is unrelated to this PR and should be removed. PRs to add a `--fix` option or to improve the linter experience are very welcome BTW :)

Reply:
Yeah lmao. Gonna revert it and see what I can do to the md linter :))))